### PR TITLE
support line-comments to struct comment

### PR DESCRIPTION
**Pull request by scorpig**
**Describe the PR**
support line-comments to struct comment

**Additional context**

image

Comments written on top of the field are not beautiful

### CODE REVIEW

File: testdata/struct_comment/web/handler.go [hunk 1]
```diff
@@ -18,5 +18,5 @@ type Post struct {
 type APIError struct {
 	// Error an Api error
 	Error     string
```
Comment (pei0804) [inline testdata/struct_comment/web/handler.go]: please add line comment. because I want to check no change.
eg: `Error     string // This is  error`

Comment (scorpig) [reply testdata/struct_comment/web/handler.go]: ok

File: testdata/struct_comment/web/handler.go [hunk 2]
```diff
@@ -16,7 +16,6 @@ type Post struct {
 }
 
 type APIError struct {
-	// Error an Api error
-	Error     string
-	CreatedAt time.Time
+	Error     string    // Error an Api error
```
Comment (pei0804) [inline testdata/struct_comment/web/handler.go]: I want to add this code.

```go
// Error an Api error
Error     string    // Error this is Line commen
```

**Comment by pei0804** [comment]
And add comment plz.
https://github.com/swaggo/swag#description-of-struct

File: testdata/struct_comment/web/handler.go (hunk 2 above)
Comment (scorpig) [reply testdata/struct_comment/web/handler.go]: QAQ

**Comment by pei0804** [comment]
OK thx.

**Comment by pei0804** [comment]
@easonlin404 
I wit for `License Compliace` sometimes.
Can you add permission for me?

**Comment by easonlin404** [comment]
would you help add this to [readme examples](https://github.com/swaggo/swag#examples) and [example code](https://github.com/swaggo/swag/tree/master/example) ?

**Comment by easonlin404** [comment]
> @easonlin404
> I wit for `License Compliace` sometimes.
> Can you add permission for me?

it seems like cannot grant permission, and I have found sometime will not trigger license scan so I have dropped check it.

**Comment by pei0804** [comment]
OK.
I add later `example code ` .

We have to decide pr flow what do you think?
eg: If add solution. required readme and example update.

**Comment by easonlin404** [comment]
It's a good idea to add example or update readme when add new feature.

**Pull request merged**